### PR TITLE
Add Spanish Language Support

### PR DESCRIPTION
**Pull request by clostao**

### CODE REVIEW

File: res/translations/es/messages.ftl [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+    Pero no te preocupes, puedes actualizarte a la red actual en un solo clic!
+upgrade_button_upgrade = Actualizar a {$chain_name}
+
+loading_title = Cargando
+loading_configuration_title = Cargando configuración
+loading_configuration_step_loading = Cargando configuración...
+loading_configuration_step_reading = Leyendo configuración...
+loading_configuration_step_configuration_exists = Leyendo configuración...
+loading_configuration_step_configuration_not_found = Leyendo configuración...
```
Comment (nazar-pc) [inline res/translations/es/messages.ftl:26]: These should change despite them being the same string in English. See the key for hints on what it should be.

File: res/translations/es/messages.ftl [hunk 2]
```diff
@@ -0,0 +1,211 @@
+welcome_title = Welcome
+welcome_message =
+    Space Acres es una aplicación gráfica (GUI) diseñada para unirte como granjero en la Red de Autonomys.
+
+    Antes de continuar, necesitas 3 cosas:
+    ✔ Una cartera donde recibirás las recompensas (utiliza Subwallet, la extensión de polkadot{”{”}.js{”}”} o cualquier otra cartera compatible con la cadena Substrate)
+    ✔ 100 GB de espacio en un SSD de buena calidad para almacenar los datos del nodo
+    ✔ Cualquier SSD (o varios) con tanto espacio como puedas permitirte para fines de agricultura, esto es lo que generará las recompensas
+
```
Comment (nazar-pc) [inline res/translations/es/messages.ftl:9]: Should be no new line hehe

**Review by nazar-pc — COMMENTED** [review]
Thanks for adding this. Please check formatting, we ensure everything is valid at compile time here.

**Comment by nazar-pc** [comment]
Also see changes in last commit of https://github.com/autonomys/space-acres/pull/288

**Comment by clostao** [comment]
> @clostao PR was open for about two weeks, do you plan to finish it?

My bad, I forgot about it

**Comment by nazar-pc** [comment]
Can you update the file locally and push a single commit instead of multiple "Update messages.ftl", please?

Each commit triggers useless email notification for me.

**Comment by nazar-pc** [comment]
I squashed all the commits, but it still doesn't compile, meaning there are issues in translation file and it doesn't match other translations (this is why we check it at compile time).

**Comment by clostao** [comment]
> I squashed all the commits, but it still doesn't compile, meaning there are issues in translation file and it doesn't match other translations (this is why we check it at compile time).

It is now solved, I've also added the two other new entries and now compiles fine.

**Review by nazar-pc — APPROVED** [review]
Awesome, thanks!

**Pull request merged**